frontend/packages/component-library/src/image/Image.tsx

```diff
@@ -46,31 +48,36 @@ const Image: React.FC<ImageProps> = ({
   className,
   onLoad,
   onError,
+  imageComponent,
```

Comment:
For consistency with codebase I think it would be better to do it like this:

```suggestion
  ImageComponent = 'img',
```